include/translate-option.php

```diff
@@ -31,21 +31,21 @@ class PLL_Translate_Option {
 	 * @since 2.9
 	 *
 	 * @param string $name Option name.
-	 * @param object $keys Recursive array of option keys to translate in the form:
-	 *   array(
-	 *     'option_key_to_translate_1' => 1,
-	 *     'option_key_to_translate_2' => 1,
-	 *     'my_group' => array(
-	 *       'sub_key_to_translate_1' => 1,
-	 *       'sub_key_to_translate_2' => 1,
-	 *     ),
-	 *   )
-	 *   Note: only keys are interpreted. Any scalar can be used as values.
-	 * @param string $args {
-	 *   Optional. Array of arguments for registering the option.
+	 * @param array  $keys Recursive array of option keys to translate in the form:
+	 *    array(
+	 *      'option_key_to_translate_1' => 1,
+	 *      'option_key_to_translate_2' => 1,
+	 *      'my_group' => array(
+	 *        'sub_key_to_translate_1' => 1,
+	 *        'sub_key_to_translate_2' => 1,
+	 *      ),
+	 *    )
+	 *    Note: only keys are interpreted. Any scalar can be used as values.
```

Comment:
This is not consistent with the parameter below.

Reply:
Could you clarify?

Reply:
I understand $args could contain a $context and a $sanitize_callback value. Am I right ?